File: 2 semestre/Técnica de Desenvolvimento de Algoritmos/Listas de Exercicios/Lista 1/ex7.c
#include <stdio.h>

main()
{
    //vars
    float pesoAtual, pesoGordo, pesoMagro;

    printf("\t\t\tComparacao de pesos\n\n");

    //Entrada
    printf("Insira seu peso atual em KG: ");
    scanf("%f",&pesoAtual);

    //Processamento
    pesoGordo = pesoAtual * 1.15;
    pesoMagro = pesoAtual * 0.8;

    //Saída
    printf("Peso se engordar 15%: %.1f KG\n", pesoGordo);
    printf("Peso se emagrecer 20%: %.1f KG\n", pesoMagro);

}

File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 1/src/Ex7.java
//Importando Scanner
import java.util.Scanner;

public class Ex7{
	
	public static void main(String[] args){
		
	//Instanciando Scanner
	Scanner ent = new Scanner(System.in);
	
	//vars
	double pesoAtual, pesoGordo, pesoMagro;
	
	
	System.out.println("\n\n");
	
	//Entrada
	System.out.print("Insira o peso atual em Kg: ");
	pesoAtual = ent.nextDouble();
	
	//Processamento
	pesoGordo = pesoAtual * 1.15;
	pesoMagro = pesoAtual * 0.8;
	
	//Saída
	System.out.printf("Peso se engordar 15%%: %.1f Kg\n", pesoGordo);
	System.out.printf("Peso se emagrecer 20%%: %.1f Kg", pesoMagro);
		
	}
}